packages/rancher-logging/generated-changes/patch/values.yaml.patch

```diff
@@ -105,6 +110,7 @@
 +global:
 +  cattle:
 +    systemDefaultRegistry: ""
++    # This value is immutable if set by the UI.
```

Comment:
**Add comment:** 

If not using the Rancher UI, then you must set `global.cattle.windows.enabled` to `true` for Windows cluster support.